src/main/webapp/3Dscript.js

```diff
@@ -0,0 +1,183 @@
+import {FlyControls} from 'https://threejs.org/examples/jsm/controls/FlyControls.js';
+import {OrbitControls} from 'https://threejs.org/examples/jsm/controls/OrbitControls.js';
+import {GUI} from 'https://threejs.org/examples/jsm/libs/dat.gui.module.js';
+
+// These are global objects.
+let scene;
+let camera;
+let renderer;
+let clock;
+let controls;
+let orientation;
+let trajectory;
+let dataLength;
+let pose;
+let poseRotation = {degrees: 0};
+let poseScalar = {multiplier: 1};
+let posePosition = {x: 0, z: 0};
```

Comment:
These three variables are highly related. Consider bundling them into some kind of pose transformation object.